src/model_repository_manager.h

```diff
@@ -320,10 +344,19 @@ class ModelRepositoryManager {
   /// \return True if the node is ready. False otherwise.
   bool CheckNode(DependencyNode* node);
 
+  /// Get the model mappings.
+  /// \return Map with model name as key, pair of repository path and full path
+  // as value.
+  const std::unordered_map<std::string, std::pair<std::string, std::string>>&
+  ModelMappings() const
+  {
+    return model_mappings_;
+  }
```

Comment:
Doesn't seem to be used anywhere, can remove this

Reply:
Done.